selfdrive/car/hyundai/values.py

```diff
@@ -12,7 +12,8 @@ class CarControllerParams:
   def __init__(self, CP):
     if CP.carFingerprint in [CAR.SONATA, CAR.PALISADE, CAR.SANTA_FE, CAR.VELOSTER, CAR.GENESIS_G70,
                              CAR.IONIQ_EV_2020, CAR.KIA_CEED, CAR.KIA_SELTOS, CAR.ELANTRA_2021,
-                             CAR.ELANTRA_HEV_2021, CAR.SONATA_HYBRID, CAR.KONA_HEV, CAR.SANTA_FE_2022, CAR.KIA_K5_2021, CAR.KONA_EV, CAR.KONA, CAR.IONIQ_HEV_2022]:
+                             CAR.ELANTRA_HEV_2021, CAR.SONATA_HYBRID, CAR.KONA_HEV, CAR.SANTA_FE_2022,
```

Comment:
seems like all new cars should go here. want to submit a PR to blacklist the 255 cars?

Reply:
> seems like all new cars should go here. want to submit a PR to blacklist the 255 cars?

@adeebshihadeh Here's the PR with the blacklist! https://github.com/commaai/openpilot/pull/23255